.gitlab-ci.yml

```diff
@@ -0,0 +1,90 @@
+image: golang:1.11
+
+cache:
+  paths:
+    - /apt-cache
+    - /go/src/github.com
+    - /go/src/golang.org
+    - /go/src/gopkg.in
+    - node_modules
+
+variables:
+  DOCKER_HOST: tcp://docker:2375/
+  DOCKER_DRIVER: overlay2
+
+services:
+  - docker:dind
+  - mongo
```

Comment:
Unsure about this one: is it needed for unit tests?

I don't think so as it seems that we are installing mongo the old way before unit tests

Reply:
Nah, we don't need it yet...